bool sim
int16 planeID
---
string error
